src/ios/CDVOrientation.m

```diff
@@ -71,12 +68,11 @@ -(void)screenOrientation:(CDVInvokedUrlCommand *)command
                 } else if (orientationMask == 2) {
                     value = [NSNumber numberWithInt:UIInterfaceOrientationPortraitUpsideDown];
                 }
+                [[UIDevice currentDevice] setValue:value forKey:@"orientation"];
```

Comment:
Remove this line value can be nil, there is a check for nil after this block and sets there
```suggestion
                [[UIDevice currentDevice] setValue:value forKey:@"orientation"];
```

Reply:
done